### ISSUE TITLE
How to extract all content of http/https traffic flows to a txt file from mitmproxy's console view 

### ISSUE THREAD

**Issue by peter087744982** [mitmproxy/mitmproxy#3020]
Dear all,

I'm using mitmproxy to realize man-in-the-middle attack in order to monitor the network behavior from my android emulator (API 23) on genymotion environment while opening some malware apps.
In the meantime, I wonder how to extract all http/https traffic flows to a txt file from mitmproxy's console view.
I've executed "sudo mitmdump -w outputfilePath" but found that the mitmdump was listening on 0.0.0.0: 8080, which is out of my expect (my proxy is established on 211.21.X.XX:8887) so that the result txt file didn't meet our requirement.
Thanks for your kindly help : )

**Comment by mhils** [comment]
Hi,

I don't really understand what your problem is. Can you maybe try to explain again?

**Comment by peter087744982** [comment]
Hi, 

thanks for your response, I'm install mitmproxy on my Ubuntu 16.04 and its works perfectly,which means I can view all HTTP and HTTPS traffic flows on my  mitmproxy console interface after executing "sudo mitmproxy -b 211.21.X.XX -p 8887" (My proxy IP is 211.21.X.XX, port is 8887)

Then, I want to save all traffic flows (POST / GET)from mitmproxy console interface into a text file so that I can analyze these logs.

But when I executed "sudo mitmdump -w ~/Desktop/mitmLog.txt" on terminal, its shows "Listening on 0.0.0.0:8080" and the contents in mitmLog.txt is not I need. 

I'm confused about this, why the mitmdump doesn't listen on 211.21.X.XX:8887 while I executed "sudo mitmdump -w ~/Desktop/mitmLog.txt". Maybe I misused mitmdump or put wrong argument : (
